src/Microsoft.Azure.SignalR.Management/RestApiProvider.cs

```diff
@@ -16,25 +16,22 @@ internal class RestApiProvider
 
         private readonly RestApiAccessTokenGenerator _restApiAccessTokenGenerator;
 
-        private readonly string _audienceBaseUrl;
-        private readonly string _serverEndpoint;
+        private readonly ServiceEndpoint _serviceEndpoint;
+
+        private string AudienceBaseUrl => _serviceEndpoint.AudienceBaseUrl;
```

Comment:
Why do we use private property?

Reply:
To avoiding allocating 2 string references (and an enum value in the next PR)

Reply:
I think it's better than using multiple private fields that belong to the same object?

There will be an additional `AuthType` field in the next PR on `serviceEndpoint`.